background.ts

```diff
@@ -325,7 +325,7 @@ Electron.ipcMain.on('k8s-integrations', async(event) => {
   event.reply('k8s-integrations', await k8smanager?.listIntegrations());
 });
 
-Electron.ipcMain.on('k8s-integration-set', async(event, name: string, newState: boolean) => {
+Electron.ipcMain.on('k8s-integration-set', async(event, name, newState) => {
```

Comment:
Just noting that the parameter types are dropped because various signatures for `Electron.ipcMain.on` are defined in `electron-ipc-d.ts`